src/app/containers/Metadata/index.jsx

```diff
@@ -58,6 +94,7 @@ const MetadataContainer = ({ metadata, promo }) => {
                   seoHeadline={promo.headlines.seoHeadline}
                   service={metadata.createdBy}
                   type={metadata.type}
+                  about={aboutTagsContent(metadata.tags.about)}
```

Comment:
We should re-use `deepGet` which is in the `helpers/` to mitigate the case where `metadata` may not have a `tags` value which would then cause an error _"cannot ready `about` of undefined"_

Reply:
`tags` does actually return as an empty object, but for best practice we shouldn't make that assumption in the code

Reply:
👍